src/hotspot/share/gc/g1/g1FullCollector.hpp

```diff
@@ -89,6 +90,8 @@ class G1FullCollector : StackObj {
 
   G1FullGCHeapRegionAttr _region_attr_table;
 
+  HeapWord*volatile *_compaction_tops;
```

Comment:
Spacing in the type is strange.  `HeapWord* volatile* _compaction_tops;` is (I think) more usual for our code.